hack/build.sh

```diff
@@ -42,7 +42,7 @@ export CGO_ENABLED=0
 MODE="${MODE:-release}"
 
 # Build terraform binaries before setting environment variables since it messes up make
-if test "${SKIP_TERRAFORM}" != y
+if test "${SKIP_TERRAFORM}" != y && ! (echo "${TAGS}" | grep -q -e 'aro' -e 'altinfra')
```

Comment:
I wasn't aware about the specific build tag `aro`, must be skipped as well?

Reply:
Yes, so I have also added the `aro` tag here:
https://github.com/openshift/installer/pull/7656/files#diff-a4f0f6b61545d617e8db370db204ea8d9fbd62f380527ef49351bf617c781e32R1-R2

ARO has a fork of the installer and does not use Terraform (using [ARO-RP](https://github.com/Azure/ARO-RP) instead). I believe they use this ARO build tag when building their fork, so the result should be that the ARO version of the installer is built without Terraform dependencies.